src/main/java/com/squareup/catalog/demo/Main.java

```diff
@@ -76,21 +73,20 @@
    */
   private static final String ARG_TOKEN = "-token";
 
+  /**
+   * Argument used to set environment. If set to sandbox, the APIs will hit the
+   * sandbox environment. If set to production, the APIs will hit the production
+   * environment.
+   */
+  private static final String ENV_FLAG = "-env";
+
   public static void main(String[] args) {
     Logger logger = new Logger.SystemLogger();
-    Main main = new Main(logger,
-        new ApplyTaxToAllIItemsExample(logger),
-        new CloneCatalogExample(logger),
-        new CreateItemExample(logger),
-        new DeduplicateTaxesExample(logger),
-        new DeleteAllItemsExample(logger),
-        new DeleteCategoryExample(logger),
-        new GloballyEnableAllItemsExample(logger),
-        new ListCategoriesExample(logger),
-        new ListDiscountsExample(logger),
-        new LocationSpecificPriceExample(logger),
-        new RetrieveCatalogObjectExample(logger),
-        new SearchItemsExample(logger));
+    Main main = new Main(logger, new CreateItemExample(logger), new DeleteAllItemsExample(logger),
```

Comment:
[Style] Any reason not to have one per line like it was before?

Reply:
Done